project/Dependencies.scala

```diff
@@ -80,6 +80,7 @@ object Dependencies {
     akkaslsProxyProtocol % "protobuf-src",
     akkaslsSdkProtocol % "compile;protobuf-src",
     akkaDependency("akka-testkit") % Test,
+    akkaDependency("akka-actor-testkit-typed") % Test,
```

Comment:
Needed since we use the SLF4J logger directly and no way to inspect those log entries with classic testkit.